spec/yaml/swiftnav/sbp/bootload.yaml

```diff
@@ -22,15 +22,40 @@ include:
   - types.yaml
 definitions:
```

Comment:
I may be wrong, but we currently don't have an assert about message IDs not overlapping in the spec. Now that we're making these migrations, it may be some easy relief to add that.